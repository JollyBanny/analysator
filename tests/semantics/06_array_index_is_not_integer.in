type arr = array [1..2] of integer;
var x: arr;

begin
    writeln(x['x']);
end.