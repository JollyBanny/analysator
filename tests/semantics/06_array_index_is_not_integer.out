[Line: 5; Column: 15]	index is not integer
